README.md

```diff
@@ -12,7 +12,8 @@ A curated list of examples related to actix.
 * [Canduma](https://github.com/clifinger/canduma) : rust authentication server boilerplate
 * [Broker](https://crates.io/crates/broker) : Real-time Zero-Code API Server
 * [Rust, Docker & GraphQL](https://github.com/jayy-lmao/rust-graphql-docker): An example of using Dataloaders, context, and a minimal docker container. 
+* [Complete Actix 2.x REST Server](https://github.com/ddimaria/rust-actix-example): Actix 2.x HTTP Server featuring multi-database support, auth/JWTs, caching, static files, app state, tests, coverage, and docker.
 
 ## Contribute
 
-Welcome to contribute !
+Welcome to contribute !
```

Comment:
Could you also drop this change? The other part looks good.

Reply:
This is done